{"category": "ham", "to_address": "Peter Krenesky <Peter@osuosl.org>", "from_address": "Dan Williams <dcbw@redhat.com>", "subject": "Re: [sugar] Error starting up sugar", "body": "On Mon, 2007-04-09 at 10:24 -0700, Peter Krenesky wrote:\n> I've been trying to get the sugar environment built and running on my ubuntu (edgy) workstation.  The only part of build-base that fails is dbus-python.  I downloaded and built the latest dbus.   At that point i built sugar and only squeak failed to build.  \n> \n> When i attempt to startup sugar i get the following errors:\n\nWhat versions of dbus-python, dbus, and python do you have?\n\nDan\n\n> \n> peter@floyd:~/wrk/sugar/sugar-jhbuild$ ./sugar-jhbuild run\n> Extended Input Devices not yet supported. Impelement it at line 625 in ../../../../hw/kdrive/src/kinput.c\n>     xkb_keycodes             { include \"xfree86+aliases(qwerty)\" };\n>     xkb_types                { include \"complete\" };\n>     xkb_compatibility        { include \"complete\" };\n>     xkb_symbols              { include \"pc(pc101)+us\" };\n>     xkb_geometry             { include \"pc(pc101)\" };\n> Could not init font path element /usr/share/fonts/X11/TTF/, removing from list!\n> Could not init font path element /usr/share/fonts/X11/OTF, removing from list!\n> Could not init font path element /usr/share/fonts/X11/CID/, removing from list!\n> ERROR:root:Error reading public key: [Errno 2] No such file or directory: '/home/peter/.sugar/default/owner.key.pub'\n> ERROR:root:Error reading private key: [Errno 2] No such file or directory: '/home/peter/.sugar/default/owner.key'\n> Traceback (most recent call last):\n>   File \"/home/peter/wrk/sugar/sugar-jhbuild/build/bin/sugar-shell\", line 38, in \n>     from view.Shell import Shell\n>   File \"/home/peter/wrk/sugar/sugar-jhbuild/build/share/sugar/shell/view/Shell.py\", line 23, in \n>     from view.home.HomeWindow import HomeWindow\n>   File \"/home/peter/wrk/sugar/sugar-jhbuild/build/share/sugar/shell/view/home/HomeWindow.py\", line 23, in \n>     from view.home.MeshBox import MeshBox\n>   File \"/home/peter/wrk/sugar/sugar-jhbuild/build/share/sugar/shell/view/home/MeshBox.py\", line 30, in \n>     from model import accesspointmodel\n>   File \"/home/peter/wrk/sugar/sugar-jhbuild/build/share/sugar/shell/model/accesspointmodel.py\", line 20, in \n>     from hardware import nmclient\n>   File \"/home/peter/wrk/sugar/sugar-jhbuild/build/share/sugar/shell/hardware/nmclient.py\", line 28, in \n>     from hardware import nminfo\n>   File \"/home/peter/wrk/sugar/sugar-jhbuild/build/share/sugar/shell/hardware/nminfo.py\", line 268, in \n>     class NMInfoDBusServiceHelper(dbus.service.Object):\n>   File \"/home/peter/wrk/sugar/sugar-jhbuild/build/share/sugar/shell/hardware/nminfo.py\", line 288, in NMInfoDBusServiceHelper\n>     @dbus.service.method(NM_INFO_IFACE, in_signature='i', out_signature='as')\n>   File \"/var/lib/python-support/python2.5/dbus/decorators.py\", line 24, in decorator\n>     in_sig = tuple(dbus_bindings.Signature(in_signature))\n>   File \"dbus_bindings.pyx\", line 84, in dbus_bindings.SignatureIter.next\n> TypeError: exceptions must be strings, classes, or instances, not type\n> \n> \n\n_______________________________________________\nSugar mailing list\nSugar@laptop.org\nhttp://mailman.laptop.org/mailman/listinfo/sugar\n\n"}